src/ggrc/settings/__init__.py

```diff
@@ -49,6 +49,12 @@
 EXTERNAL_HELP_URL = os.environ.get("GGRC_EXTERNAL_HELP_URL")
 EXTERNAL_IMPORT_HELP_URL = os.environ.get("GGRC_EXTERNAL_IMPORT_HELP_URL")
 SERVER_SOFTWARE = os.environ.get('SERVER_SOFTWARE', '')
+ISSUETRACKER_COMPONENT_ID = os.environ.get('ISSUETRACKER_COMPONENT_ID')
+ISSUETRACKER_ISSUE_HOTLIST_ID = os.environ.get('ISSUETRACKER_ISSUE_HOTLIST_ID')
+ISSUETRACKER_HOTLIST_ID = os.environ.get('ISSUETRACKER_HOTLIST_ID')
+ISSUETRACKER_ISSUE_COMPONENT_ID = os.environ.get(
+    'ISSUETRACKER_ISSUE_COMPONENT_ID')
```

Comment:
[nit] I suggest to replace it (because of our python code guidelines) with the code u see below

```suggestion
    'ISSUETRACKER_ISSUE_COMPONENT_ID'
)
```